### PR TITLE
Bug #55677 [LiconKA] Function-4 : History - Bug

### PR DESCRIPTION
**Pull request by liconrepo**
Bug-1: Item history/view status not found, need to resolve (solved)
Bug-2: Published date was null fetched from sqlite history (solved)

### CODE REVIEW

**Comment by phamxuanlu** [comment]
OK

File: app/src/main/java/com/licon/rssfeeds/data/model/FeedItem.java [hunk 1]
```diff
@@ -106,11 +106,11 @@ public void setMediaSize(Long mediaSize) {
...
 
-    public String getViewStatus() {
-        return viewStatus;
+    public boolean isHistory() {
+        return isHistory;
     }
 
-    public void setViewStatus(String viewStatus) {
-        this.viewStatus = viewStatus;
+    public void setIsHistory(boolean isHistory) {
```
Comment (nvmanh) [inline app/src/main/java/com/licon/rssfeeds/data/model/FeedItem.java]: name convention. setHistory not setIsHistory

File: app/src/main/java/com/licon/rssfeeds/data/model/FeedItem.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
     private String author;
     private Date publicationDate;
     private Long mediaSize;
-    private String viewStatus;
+    private boolean isHistory;
```
Comment (nvmanh) [inline app/src/main/java/com/licon/rssfeeds/data/model/FeedItem.java]: should be history only

**Comment by nvmanh** [comment]
OK!

**Pull request merged**